lib/code_corps/model/github_issue.ex

```diff
@@ -34,12 +35,15 @@ defmodule CodeCorps.GithubIssue do
   def create_changeset(struct, params) do
     struct
     |> changeset(params)
-    |> cast(params, [:github_repo_id])
+    |> cast(params, [:github_pull_request_id, :github_repo_id])
+    |> assoc_constraint(:github_pull_request)
     |> assoc_constraint(:github_repo)
   end
 
   def update_changeset(struct, params) do
     struct
     |> changeset(params)
+    |> cast(params, [:github_pull_request_id])
+    |> assoc_constraint(:github_pull_request)
```

Comment:
I update the pull request ID here, too, since it's conceivable that we could receive an issue that does not have the pull request synced, but I am not positive this is true. It may be worth writing tests for this and removing its reliance, or keeping it there as a safety.